api/graphql/mutations/index.js

```diff
@@ -132,7 +132,22 @@ export function removeSkill (userId, skillIdOrName) {
   .then(() => ({success: true}))
 }
 
-export function flagInappropriateContent (userId, { category, reason, link }) {
+export function flagInappropriateContent (userId, { category, reason, linkData }) {
+  let link
```

Comment:
add some tests for this please -- if you want to pull it out into its own function and just test that instead of going through FlaggedItem.create each time, that'd be good too. (if so, might want to move the two functions into `api/graphql/mutations/flagging.js`

Reply:
@levity done